infra.bs

```diff
@@ -739,6 +732,16 @@ ordered sets; implementations can optimize based on the fact that the order is n
 from the set that matches a given condition with the given <a for=set>item</a> and remove the other
 matching items, or do nothing if none match.
 
+<hr>
+
+<p><dfn export lt="the range">The range</dfn> <var>n</var> to <var>m</var>, inclusive, creates a new
+<a>ordered set</a> containing all of the integers from <var>n</var> up to and including <var>m</var>
+in consecutively increasing order, as long as <var>m</var> is greather than or equal to
+<var>n</var>.
+
+<p class=example id=example-the-range><a for=set>For each</a> <var>n</var> in <a>the range</a> 1 to
+4, inclusive, &hellip;
```

Comment:
If we can use this for all ranges we define anywhere we could maybe start leaving it out, but then phrases like "is an ASCII whitespace" become somewhat ill-defined. So I figured I'd stay consistent with those uses, but not actually start linking them.